src/db/DB_Table_Currencyformats_V1.h

```diff
@@ -173,7 +173,7 @@ struct DB_Table_CURRENCYFORMATS_V1 : public DB_Table
         db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS_V1 VALUES ('62', '%s', 'L', '', '.', ' ', '', '', '100', '1', 'HNL')", wxTRANSLATE("Honduran lempira")));
         db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS_V1 VALUES ('63', '%s', 'HK$', '', '.', ' ', '', '', '100', '1', 'HKD')", wxTRANSLATE("Hong Kong dollar")));
         db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS_V1 VALUES ('64', '%s', 'Ft', '', '.', ' ', '', '', '1', '1', 'HUF')", wxTRANSLATE("Hungarian forint")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS_V1 VALUES ('65', '%s', 'kr', '', '.', ' ', '', '', '1', '1', 'ISK')", L"Icelandic króna"));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS_V1 VALUES ('65', '%s', 'kr', '', '.', ' ', '', '', '1', '1', 'ISK')", L"_tr_Icelandic króna"));
```

Comment:
`L"_tr_Icelandic króna"` is wrong
Should be `wxTRANSLATE(L"Icelandic króna")` IMHO

Reply:
Both are incorrect.
This trik with '_tr_' prefix is result of bug   #739 fix. I think is better to use only Latin names for currencies.
```wxTRANSLATE("Icelandic krona")```

Reply:
Please check out the latest fix as generated by sqlite2cpp.py